src/mongocxx/test/options/create_collection.cpp

```diff
@@ -0,0 +1,56 @@
+#include "catch.hpp"
+
+#include <bsoncxx/document/value.hpp>
+#include <bsoncxx/document/view.hpp>
+#include <bsoncxx/types.hpp>
+#include <bsoncxx/types/value.hpp>
+#include <mongocxx/options/create_collection.hpp>
+
+using namespace bsoncxx;
+using namespace mongocxx;
+
+TEST_CASE("create_collection", "[create_collection]") {
+    options::create_collection cc;
+
```

Comment:
The CHECK_OPTIONAL_ARGUMENT macro requires that these methods have setters and getters defined.  I haven't implemented getters because we have the to_document() method.  For now I'd like to leave the getters off, we can always add them later.